### PR TITLE
Add missing output columns

### PR DESCRIPTION
**Pull request by nachoalonsoportillo**

### CODE REVIEW

**Comment by Court72** [comment]
@shsagir

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/kusto/management/show-table-details-command.md [hunk 1]
```diff
@@ -38,17 +38,19 @@ Requires [Database viewer permission](./access-control/role-based-access-control
 | `ShardingPolicy`           | String   | The table's effective`*` sharding policy, serialized as JSON.                                   |
 | `MergePolicy`              | String   | The table's effective`*` merge policy, serialized as JSON.                                      |
 | `StreamingIngestionPolicy` | String   | The table's effective`*` streaming ingestion policy, serialized as JSON.                        |
+| `IngestionBatchingPolicy`  | String   | The table's effective`*` batching ingestion policy, serialized as JSON.                         |
 | `MinExtentsCreationTime`   | DateTime | The minimum creation time of an extent in the table (or null, if there are no extents)`**`.     |
 | `MaxExtentsCreationTime`   | DateTime | The maximum creation time of an extent in the table (or null, if there are no extents)`**`.     |
 | `RowOrderPolicy`           | String   | The table's effective row order policy, serialized as JSON.                                     |
+| `TableId`                  | String   | The table's internal GUID identifier.                                                           |
```
Comment (shsagir) [inline data-explorer/kusto/management/show-table-details-command.md:45]:
```suggestion
| `TableId`                  | String   | The table's unique ID.                                                           |
```

**Review by shsagir — APPROVED** [review]

**Comment by shsagir** [comment]
@nachoalonsoportillo Thank you for your contribution. We've submitted a documentation change based on your suggestion, and the updated article should be live by tomorrow.

#sign-off

**Comment by shsagir** [comment]
#sign-off

**Pull request merged**